documents/service.go

```diff
@@ -49,8 +48,8 @@ type Service interface {
 	// CreateProofsForVersion creates proofs for a particular version of the document given the fields
 	CreateProofsForVersion(ctx context.Context, documentID, version []byte, fields []string) (*DocumentProof, error)
 
-	// RequestDocumentSignature Validates and Signs document received over the p2p layer
-	RequestDocumentSignature(ctx context.Context, model Model) (*coredocumentpb.Signature, error)
+	// ReceiveDocumentSignatureRequest Validates and Signs document received over the p2p layer
+	ReceiveDocumentSignatureRequest(ctx context.Context, model Model, sender identity.DID) (*coredocumentpb.Signature, error)
```

Comment:
`RequestDocumentSignature` makes sense here since the intention is to get signature. `ReceiveDocumentSignatureRequest` implies SignatureRequest is being received. This is not what implementation is doing. Lets revet this pls

Reply:
I renamed it exactly because of the confusion in your comment this is actually receiving it

Reply:
`ReceiveDocumentSignatureRequest(ctx context.Context, model Model, sender identity.DID) (*coredocumentpb.Signature, error)`

The function is actually returning the signature. It is not receiving, no ?